stactools_spot/requirements.txt

```
@@ -0,0 +1,2 @@
+fiona==1.8.13
+boto3==1.11.9
```

```suggestion
boto3==1.17.40
```

`stactools_threedep` [requires a newer version of boto3](https://github.com/stac-utils/stactools/blob/master/stactools_threedep/requirements.txt#L1) and pip dependency resolution was [struggling with the conflicting requirements](https://github.com/stac-utils/stactools/runs/2718764779).